src/containers/articles/Article.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Skeleton } from 'antd';
+import { getArticle, deleteArticle } from '../../actions/articleActions';
+import Article from '../../components/articles/Article';
+
+class ArticlePage extends Component {
+  componentDidMount() {
+    const { articleData, getArticle, history } = this.props; // eslint-disable-line
+    if (!articleData.id) {
+      const slug = history.location.pathname.split('/')[2];
+      getArticle(slug);
+    }
+  }
+
+  handleEditClick = () => {
+    const { articleData, getArticle, history } = this.props; // eslint-disable-line
+    getArticle(articleData.slug);
+    history.push('/articles/update');
```

Comment:
This action `history.push('/articles/update');` should happen after this action `getArticle(articleData.slug);
`  is successful. This logic has not been handled correctly here.